custom/abt/reports/late_pmt.py

```diff
@@ -84,6 +84,13 @@ class LatePmtReport(GenericTabularReport, CustomProjectReport, DatespanMixin):
     report_title = "Late PMT"
     slug = 'late_pmt'
     name = "Late PMT"
+    is_support_translations = True
+
+    languages = (
+        'en',
+        'fra',
+        'por'
+    )
```

Comment:
In general would we ever set `languages` to something without also setting `is_support_translations` to `True`? If not, we might be able to just replace `is_support_translations` with

```
@property
def supports_translations(self):
    return bool(self.languages)
```

Reply:
We can't use ```@property``` because in the place where we use this we have only the class type. So all methods with ```@property``` annotation are not evaluated.